### ISSUE TITLE
Card items overflow out of cards

### ISSUE THREAD

**Issue by mtlaso** [codecrafters-io/frontend#1768]
## Steps
1. Visit someone's profile (https://app.codecrafters.io/users/rohitpaulk)
2. Look into the challenges section
3. Some icons are overflowing out of their card (when a challenge has been solved with multiple languages)

(firefox)
![firerfox](https://github.com/codecrafters-io/frontend/assets/29934021/4178e4e3-f35f-46c7-88c8-51db800f6edb)

(chromium)
![chrome](https://github.com/codecrafters-io/frontend/assets/29934021/0463c723-9e4f-436e-9c2d-7d0b4df0d427)

## Proposed solution
Make sure the icons don't overflow

**Comment by rohitpaulk** [comment]
Spec for solution: let's pick upto 3 languages. Prioritize based on this ordered list: 

- Completed languages, sorted by last activity
- Incomplete languages, sorted by last activity

**Comment by yuvrajkarna2717** [comment]
"Hi @rohitpaulk, is this issue still relevant to work on? If yes, could you provide more details or context to help improve it?

**Comment by rohitpaulk** [comment]
@yuvrajkarna2717 yep! We should limit to 3 icons, you'll find the spec above here: https://github.com/codecrafters-io/frontend/issues/1768#issuecomment-2255667285

**Comment by Wassaf001** [comment]
Hey, what should I add in .env, backend url?  

# This should match the URL defined in `core`.
BACKEND_URL="https://<your-name>-backend.ccdev.dev"

**Comment by ryan-gang** [comment]
You can use our staging server to develop against. 
`BACKEND_URL="https://backend-staging.codecrafters.io"`

**Comment by Wassaf001** [comment]
ok thanks

**Comment by rohitpaulk** [comment]
Added this to our default `.env.example` here: https://github.com/codecrafters-io/frontend/pull/2639

**Comment by Wassaf001** [comment]
I am wondering, how can I check my code is working or not, as I cannot reproduce the bug.

Any help please.

**Comment by rohitpaulk** [comment]
@Wassaf001 you can try editing this file to mimic the scenario: https://github.com/codecrafters-io/frontend/blob/main/tests/acceptance/view-user-profile-test.js. You can add `pauseTest()` [as described here](https://guides.emberjs.com/release/testing/testing-application/) to see the UI in the middle of a test.

**Comment by Wassaf001** [comment]
ok thank you.

**Comment by Wassaf001** [comment]
According to the spec mentioned here: 

> Spec for solution: let's pick upto 3 languages. Prioritize based on this ordered list:
> 
> * Completed languages, sorted by last activity
> * Incomplete languages, sorted by last activity

<img width="1421" alt="Image" src="https://github.com/user-attachments/assets/c3be5bd3-1133-4ec6-900a-a4dd9c8278d7" />

<img width="981" alt="Image" src="https://github.com/user-attachments/assets/f608b5a4-fe8d-471d-9377-2f5b472070dc" />

This is my progress. 

Is it ready for a PR?

**Comment by rohitpaulk** [comment]
@Wassaf001 one thing I'm noticing is that in the first screenshot I have three completed languages - PHP, TS and Kotlin, but in the second it only shows 2 (would have expected 3). 

Would recommend opening a PR anyway! Easier to share feedback when we have concrete code to look at.

**Comment by Wassaf001** [comment]
ok will do that.

**Comment by Wassaf001** [comment]
I have opened a PR, what is the process further, if any change or improvement is required let me know, I will do it.
